src/pages/reciters/index.tsx

```
@@ -15,8 +15,9 @@ import QuranReciterListHero from '@/components/Reciter/QuranReciterListHero';
 import RecitersList from '@/components/Reciter/RecitersList';
 import { getAllChaptersData } from '@/utils/chapter';
 import { getLanguageAlternates } from '@/utils/locale';
-import { getCanonicalUrl } from '@/utils/navigation';
-import { REVALIDATION_PERIOD_ON_ERROR_SECONDS } from '@/utils/staticPageGeneration';
+import { getCanonicalUrl, getRecitersNavigationUrl } from '@/utils/navigation';
```

ESLint

[error] 'getRecitersNavigationUrl' is defined but never used.

(@typescript-eslint/no-unused-vars)